gcc/multiple_target.cc

```diff
@@ -323,6 +323,19 @@ expand_target_clones (struct cgraph_node *node, bool definition)
       return false;
     }
 
+  /* No need to clone when AFMV option is provided, produce error */
+
+  // extern bool global_afmv_enabled; // This should be defined where handles command-line options
+     bool global_afmv_enabled = true; /* Assuming AFMV option is provided here for testing */
```

Comment:
Indentation should be adjusted for the variable so it matches the if statement below.